.eslintrc.json

```diff
@@ -17,5 +17,6 @@
     },
     "rules": {
         "no-plusplus": "off"
+        
```

Comment:
Extra white line for no reason.
Seems like leftover form a change, I think you can delete this.